generate/beat/{{cookiecutter.beat}}/beater/{{cookiecutter.beat}}.go

```diff
@@ -13,42 +13,29 @@ import (
 )
 
 type {{cookiecutter.beat|capitalize}} struct {
-	config config.Config
-	done   chan struct{}
-	client publisher.Client
+	done       chan struct{}
+	config     config.Config
```

Comment:
build is all greenish. consider adding a 'go fmt' check. Maybe it's some good practice to call `go fmt` as post-processing step to ensure all files generated are in good shape.